crates/bevy_ecs/src/error/handler.rs

```diff
@@ -1,14 +1,6 @@
-use crate::{component::Tick, error::BevyError, resource::Resource};
-use alloc::borrow::Cow;
+use crate::{error::BevyError, resource::Resource};
 
-/// Additional context for a failed system run.
-pub struct SystemErrorContext {
```

Comment:
I believe the rationale here was that SystemErrorContext is a "dependency of an error handler". It exists for and is defined by the "error handler context", so it belongs either in the error handler module, or as a submodule (and by homing it next to the handlers, it gives readers of that code the knowledge of what is being passed in).

Reply:
(I didn't write that code, but that was my read of the situation)

Reply:
That makes sense to me. My rationale here was "make handler.rs ignorable", by making it a simple collection of premade error handlers. I'll just revert this though.